cauldron/redis_cache.py

```diff
@@ -107,6 +107,22 @@ def get_key(cls, key, namespace=None):
                 key = cls._get_key(namespace, key)
             return (yield from redis.get(key, encoding=cls._utf8))
 
+    @classmethod
+    @coroutine
+    def set_key_sadd(cls, key, value, namespace=None):
```

Comment:
We can add support for adding multiple members so that in future if multiple members addition is required we shouldn't change this method or write a new method.
So, we will take values and send values to redi's sadd